eox_core/settings/production.py

```diff
@@ -141,7 +145,7 @@ def plugin_settings(settings):  # pylint: disable=function-redefined
             integrations=[
                 DjangoIntegration(),
             ],
-
+            **sentry_extra_options,
```

Comment:
can we move this to line 151?

Reply:
what if `sentry_extra_options` is invalid?

Reply:
What would be an invalid `sentry_extra_option`? What cases should I cover?

Reply:
An option that's not handled by sentry_sdk.init, or does the method just ignores the options that don't recognize?